### PR TITLE
miniserve: add page

### PR DESCRIPTION
**Pull request by K-REBO**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/common/miniserve.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+> For when you really just want to serve some files over HTTP right now!.
+> More information: <https://github.com/svenstaro/miniserve>.
+
+- Serve a directory:
+
+`miniserve {{path/to/directory}}`
+
+- Serve a single file:
+
+`miniserve {{path/to/source_file}}`
```
Comment (navarroaxel) [inline pages/common/miniserve.md:13]:
```suggestion
`miniserve {{path/to/file}}`
```

File: pages/common/miniserve.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+- Require username/password:
```
Comment (navarroaxel) [inline pages/common/miniserve.md:15]:
```suggestion
- Serve a directory using HTTP basic authentication:
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]
Hi @K-REBO, welcome and thanks for your submission. I left some comments.

File: pages/common/miniserve.md (hunk 1 above)
Comment (K-REBO) [reply pages/common/miniserve.md:13]: Thank you for good suggestion.
It's looks so good.

**Review by navarroaxel — APPROVED** [review]

File: pages/common/miniserve.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
+# miniserve
+
+> Simple HTTP file server CLI.
+> For when you really just want to serve some files over HTTP right now!.
```
Comment (CleanMachine1) [inline pages/common/miniserve.md:4]: I personally, don't think this is necessary.

Comment (K-REBO) [reply pages/common/miniserve.md:4]: Should I delete this line?

Comment (sbrl) [reply pages/common/miniserve.md:4]: Hey there! Yeah, I think this extra line isn't really necessary :-)

**Review by CleanMachine1 — APPROVED** [review]

File: pages/common/miniserve.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
+# miniserve
+
+> Simple HTTP file server CLI.
+> More information: <https://github.com/svenstaro/miniserve>.
+
+- Serve a directory:
```
Comment (marchersimon) [inline pages/common/miniserve.md:6]: Is `serve` a common word? I assume it means something like `make accessible`.
What do you think about
```suggestion
- Make a directory accessible over HTTP:
```

Comment (sbrl) [reply pages/common/miniserve.md:6]: Yeah, `Serve` is a commonly used word for this. In fact, there's a Node.js package called `serve` that does something very similar to this command.

**Review by sbrl — APPROVED** [review]
Thanks, @K-REBO!

**Review by marchersimon — APPROVED** [review]
Looks good, thanks

**Pull request merged**